xmtp_mls/src/groups/group_mutable_metadata.rs

```diff
@@ -71,6 +69,18 @@ impl fmt::Display for MetadataField {
     }
 }
 
+#[derive(Default, Debug, Clone, PartialEq)]
+pub struct ConversationMessageDisappearingSettings {
```

Comment:
small nit: WDYT of renaming this to something more concise, maybe `MessageExpiryConfig`
instead of `ConversationMessageDisappearingSettings`?

Reply:
Agree the naming is quite long, still didn't want to use a name that has other meanings, in my mind expired messages could be something else in other scenarios! wdyt?
for the documentations, surelly will do that.